docker/synapse3_10.Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG SYTEST_IMAGE_TAG=bookworm
+FROM matrixdotorg/sytest:bookworm
+
+RUN apt-get -qq update && apt-get -qq install -y \
+    python3.10 python3.10-dev python3.10-venv eatmydata \
+    redis-server
```

Comment:
one thought that occurs to me: if we add an `ARG PYTHON_VERSION=python3` to `synapse.Dockerfile`, and replace each reference to `python3` with `${PYTHON_VERSION}` ... then we can replace this whole dockerfile with a build arg of `PYTHON_VERSION=python3.10`.

wdyt?

Reply:
I think that's a more sensible approach than the one I took-I'll give using build args a shot!